src/popup/router/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -6,6 +6,18 @@
   >
     <TransactionOverview :tx="transaction" />
 
+    <DetailsItem
+      v-if="getNameFee(transaction)"
+      :label="$t('modals.confirm-transaction-sign.nameFee')"
+      class="name-fee"
+    >
+      <TokenAmount
+        slot="value"
+        :amount="getNameFee(transaction)"
+        data-cy="name-fee"
```

Comment:
We are using data-cy only for the tests. There are no tests associated with data-cy="name-fee". Why do you introduce it?